src/components/Icon.js

```diff
@@ -53,6 +53,6 @@ Icon.defaultProps = {
   width: normalize(24),
   height: normalize(24),
   // size is for font icon
-  size: normalize(26),
+  size: normalize(44),
```

Comment:
this is not what we need.

we need to increase the touchable size, not the visible size. (see share icon in screenshot)

also this does not affect all icons equally (xml vs non xml).

![Screenshot 2021-06-14 at 11 01 09](https://user-images.githubusercontent.com/59824597/121867123-e4115500-ccff-11eb-84fe-a55f84e0a3c2.png)

Reply:
right! how about increasing the [hitslop](https://reactnative.dev/docs/pressable#hitslop) of the icons ?

Reply:
solved in 879b31d